server/schemas.py

```diff
@@ -10,11 +13,38 @@ class ChatEvent(Event):
     """Sent by the client when they wish to chat with the rest of the server."""
 
     type: Literal["chat"]
+    player_name: str
     chat_message: str
 
+    def to_response(self):
```

Comment:
Instead of using `to_response` here, why not just remove the `player_name` field, and have a single `chat_message` field? And use the `.json` method pydantic already provides. The schemas should ideally represent what the JSON looks like. Another option I can think of is keeping `player_name` and `chat_message`, not having a custom `to_response`, and leave it to the client to show the usernames next to the message (doing the `username: message`) thing.